app/modules/net.c

```diff
@@ -121,12 +121,12 @@ static void net_socket_disconnected(void *arg)    // tcp only
   if(nud->pesp_conn)
     c_free(nud->pesp_conn);
   nud->pesp_conn = NULL;  // espconn is already disconnected
-  lua_gc(gL, LUA_GCSTOP, 0);
+  lua_gc(L, LUA_GCSTOP, 0);
```

Comment:
As a matter of interest, why is the GC disabled here for a short time? I don't think that it needs to be..... [Worse, if it does need to be disabled, then the disable call is probably missing from lots of other places as well]